tests/5.1/error/test_error_message.c

```diff
@@ -0,0 +1,34 @@
+//===--- test_error_message.c ----------------------------===//
+//
+//  OpenMP API Version 5.1 Aug 2021
+//
+//  Test uses error directive with message clause to print a message,
+//  if error condition is reached then "runtime error!" is printed,
+//  otherwise is simply used to print a message. 
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+int errors, i;
+
+int test_error_message() {
+    char msg[] = "Success! Error message works properly"
+    i = 0;
+    printf("Test should print a 'Success!' error message: \n");
+    #pragma omp error message(msg)
+    i+=5;
+    OMPVV_WARNING_IF("Error directive caused runtime error", i!=5);
```

Comment:
I am not sure what this test is doing. If there is a RT error the test will terminate at 23 and not reach 25.